target/linux/loongarch64/config-6.6

```diff
@@ -242,7 +242,6 @@ CONFIG_FB_CORE=y
 CONFIG_FB_DEFERRED_IO=y
 CONFIG_FB_DEVICE=y
 CONFIG_FB_EFI=y
-CONFIG_FB_IOMEM_FOPS=y
 CONFIG_FB_IOMEM_HELPERS=y
```

Comment:
I would suggest not touching these symbols. They should not be manually edited.

Reply:
Fixed